examples/cifar10_cnn.py

```diff
@@ -27,6 +28,12 @@
 save_dir = os.path.join(os.getcwd(), 'saved_models')
 model_name = 'keras_cifar10_trained_model.h5'
 
+
+def top_3_acc(y_true, y_pred):
+    '''Example of how to use custom metrics - Shows accuracy when looking at
+    the top 3 predictions.'''
+    return top_k_categorical_accuracy(y_true, y_pred, k=3)
```

Comment:
I don't think this is useful in this example. Please remove.

Reply:
Sure